File: Submissions/#0516_div2/A/main.cpp
#include <bits/stdc++.h>
using namespace std;

int main() {

  vector<int> abc(3);
  cin >> abc[0] >> abc[1] >> abc[2];

  sort(abc.begin(), abc.end());

  if (abc[2] < abc[0]+abc[1]) {
    cout << 0 << endl;
  } else {
    cout << abc[2] - abc[0] - abc[1] + 1 << endl;
  }
}

File: Submissions/#0516_div2/A/main.java
import java.util.*;

public class Main {

  public static void main (String[] args) {

    Scanner sc = new Scanner(System.in);
    int[] abc = new int[3];
    abc[0] = sc.nextInt();
    abc[1] = sc.nextInt();
    abc[2] = sc.nextInt();
    sc.close();

    Arrays.sort(abc);

    if (abc[2] < abc[0]+abc[1]) {
      System.out.println(0);
    } else {
      System.out.println(abc[2] - abc[0] - abc[1] + 1);
    }

  }

}